qiskit_addon_obp/backpropagation.py

```diff
@@ -164,7 +164,8 @@ def handle_timeout(signum, frame):
             # PERF: we will likely need to parallelize this loop
             for i in range(num_observables):
                 non_trivial_slice = False
-                for op_idx, op_node in enumerate(circuit_to_dag(slice_).topological_op_nodes()):
+                op_nodes = list(circuit_to_dag(slice_).topological_op_nodes())[::-1]
```

Comment:
@mrossinek I'll just cast to list here and reverse it. That way I can get its length for logger

Reply:
I guess that is fine. Although I think it would have been fine if the logger counted forwards

Reply:
Well reversed cant act on a generator, so it was going to need to be cast either way in order to iterate backwards. We can adjust the logger before release if you want it to behave differently

Reply:
I was thinking we could do:
```
enumerate(reversed(...))
```
But as I said, it is not a big deal :+1: